common_docs_zh/ecosystem/fluentbit.md

```diff
@@ -0,0 +1,369 @@
+---
+{
+"title": "Fluent Bit Doris Output Plugin",
+"language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Fluent Bit Doris output plugin
+
+[Fluent Bit](https://fluentbit.io/) 是一个快速的日志处理器和转发器，它支持自定义输出插件将数据写入存储系统，Fluent Bit Doris output plugin 是输出到 Doris 的插件。
+
+Fluent Bit Doris output plugin 调用 [Doris Stream Load](../data-operate/import/stream-load-manual.md) HTTP 接口将数据实时写入 Doris，提供多线程并发，失败重试，自定义 Stream Load 格式和参数，输出写入速度等能力。
+
+使用 Fluent Bit Doris output plugin 主要有三个步骤：
+1. 下载或编译包含 Doris output plugin 的 Fluent Bit 二进制程序
+2. 配置 Fluent Bit 输出地址和其他参数
+3. 启动 Fluent Bit 将数据实时写入 Doris
+
+## 安装（alpha）
+
+### 从官网下载
+
+https://github.com/joker-star-l/fluent-bit/releases/download/v3.1.9-alpha/fluent-bit
+
+### 从源码编译
+
+克隆 https://github.com/joker-star-l/fluent-bit 的 dev 分支，在 build/ 目录下执行
+
+```
+cmake -DFLB_RELEASE=ON ..
+make
+```
+
+编译产物为 build/bin/fluent-bit。
+
+## 参数配置
+
+Fluent Bit Doris output plugin 的配置如下：
+
+配置 | 说明
```

Comment:
it's recommended you to use Chat GPT double check the Table format

Reply:
I double checked it. The format is right. Are there any other problems?